src/main/webapp/app/about-page/about-page.template.html

```diff
@@ -3,19 +3,21 @@
     <md-card>
       <md-card-title>
         <md-card-title-text>
-          <span class="md-headline">About page card title</span>
+          <span class="md-headline">About this project</span>
         </md-card-title-text>
       </md-card-title>
       <md-card-content>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-        Pellentesque sollicitudin nunc arcu, in aliquam sem porta lacinia. Morbi semper ex ac felis vulputate, id vestibulum sapien pellentesque.
-        Praesent odio risus, euismod condimentum nisl ac, sodales accumsan nibh.
-        Vestibulum aliquam ex sed bibendum pulvinar. Ut eleifend mi nec lacinia faucibus.
-        In accumsan purus dui, nec consequat elit mollis vitae. Vestibulum mollis blandit massa a mollis.
-        Nam semper suscipit ante in dignissim. Vivamus non consectetur sapien, ut rhoncus libero.
-        Morbi justo ipsum, cursus in quam eget, varius vulputate leo. Ut quis quam sit amet orci rhoncus ornare. Duis vitae magna elit.
-        Fusce odio sapien, pharetra ac sodales a, lacinia vel erat. Morbi ut velit eu risus accumsan ornare.
+        <p>This a training project we are working on at IRI Greece (ACE dpt) for our 6-month long internship.</p>
```

Comment:
Please change to "we are working at IRI’s Analytics Centre of Excellence (ACE)... "